### DISCUSSION TITLE
How to add a corporate mentor to the cover？

### DISCUSSION THREAD

**Discussion by yapple** [ustctug/ustcthesis#310] [Q&A]
I have set the degree-type to professional like this, and it works.
`\documentclass[degree=master,degree-type=professional]{ustcthesis}`
How to add a corporate mentor to the cover？

**Comment by zepinglee** [comment]
> I have set the degree-type to professional like this, and it works.
> `\documentclass[degree=master,degree-type=professional]{ustcthesis}`
> How to add a corporate mentor to the cover？

The format of supervisor is not clearly specified in the *Guide*. You may set both names of supervisors as `supervisor*        = {Prof. XXX, Prof. XXX},`。
